timeseries/tests/unittests/common.py

```diff
@@ -51,7 +51,7 @@ def get_data_frame_with_item_index(
     )
 
 
-DUMMY_TS_DATAFRAME = get_data_frame_with_item_index(["A", "B", "C", "D"])
+DUMMY_TS_DATAFRAME = get_data_frame_with_item_index(["10", "A", "2", "1"])
```

Comment:
Change the items to not be sorted alphabetically, this allows us to catch bugs related to changed item order.